### ISSUE TITLE
Regression bug from 1.1.13 to 1.2.x ?

### ISSUE THREAD

**Issue by felixchenier** [Chilipp/autodocsumm#33]
Hello

I have a problem with autodocsumm 1.2.x which generates a summary of all the module objects, (wrongly) including the imported objects. Also, the links between the summary table and the objects are not all created (some summary entries whose links worked before are not links at all in 1.2.x).

Reverting to 1.1.13 resolved the case for me.

I'm sorry I don't have a simple example project to reproduce the issue, but at least I'll initiate the bug report :-)

Don't hesitate if you're not able to reproduce the bug, I'll try to find time to build a simple projet.

Thank you anyways for this very helpful tool.

**Comment by Chilipp** [comment]
Hey @felixchenier ! Thanks for reporting. Something must go won't with the following of members... It's likely also what causes the problem in #32, as the function that we talk about there is imported and actually would not have been listed...

I am using the same filtering as autodoc, but likely there has been introduced a bug in the last version, sorry. I need to have a closer look.

**Comment by bouweandela** [comment]
Note that the version numbers above and in the title are off by 1 in the major version number, it's 0.1.13 and 0.2.x

**Comment by felixchenier** [comment]
Good morning Philipp

Thank you for working on this issue. I tried 0.2.2 today and I saw some difference but I don't think the issue is solved yet. Some imports such as imports from typing (Dict, Tuple, etc.) are correctly undocumented now, but some other imports (from my own libraries but also from other standard libraries such as copy) are. Also, I still get no link in the generated table.

I joined two screenshot to illustrate and maybe help you as much as I can. Both screenshots use the exact same configuration, but two different versions of autodocsumm. I hope it can be helpful.

Thank you again for the great work!

Felix

![Screen Shot 2021-01-08 at 10 37 14](https://user-images.githubusercontent.com/34967663/104033963-42a93480-519e-11eb-87e7-94145b227fa6.png)

![Screen Shot 2021-01-08 at 10 33 59](https://user-images.githubusercontent.com/34967663/104034003-4dfc6000-519e-11eb-8b98-5253e168ee32.png)
